server/src/packages/rwcds-form/rwcds-form.service.ts

```diff
@@ -132,17 +132,24 @@ export class RwcdsFormService {
       rwcdsForm.dcp_projectsitedescription === null
       || rwcdsForm.dcp_proposedprojectdevelopmentdescription === null
     ) {
-      const {
-        dcp_projectdescriptionproposedarea,
-        dcp_projectdescriptionproposeddevelopment,
-      } = await this.pasFormService.getLatestPasForm(_dcp_projectid_value);
-
-      if (rwcdsForm.dcp_projectsitedescription === null) {
-        rwcdsForm.dcp_projectsitedescription = dcp_projectdescriptionproposedarea;
-      }
-
-      if (rwcdsForm.dcp_proposedprojectdevelopmentdescription === null) {
-        rwcdsForm.dcp_proposedprojectdevelopmentdescription = dcp_projectdescriptionproposeddevelopment;
+     const latestPasForm = await this.pasFormService.getLatestPasForm(_dcp_projectid_value);
+
+     if (latestPasForm) {
```

Comment:
Here, the consuming function of `getLatestPasForm` checks for if the return value is `null`